install.sh

```diff
@@ -7,29 +7,30 @@ if [[ -n "$MSYSTEM" ]]; then
   exit 1
 fi
 
-source "$(dirname $0)/install/_lib.sh" # does a `cd .../install/`, among other things
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+source "$SCRIPT_DIR/install/_lib.sh"
 
 # Pre-flight. No impact yet.
-source parse-cli.sh
-source detect-platform.sh
-source dc-detect-version.sh
-source error-handling.sh
+source "$PROJECT_ROOT/install/parse-cli.sh"
```

Comment:
Can we perform a cd into the install directory here without it interfering with the functionality of this PR? That should resolve that issue of reporting the entire path

Reply:
> Can we perform a cd into the install directory here without it interfering with the functionality of this PR?

Avoiding this `cd` is the entire point of this PR, it messes with the Docker build context. The integration tests did not catch this problem, because the `cd` only happens when actually running `install.sh`.

If you prefer relative paths, we could try an alternative solution I proposed in https://github.com/getsentry/self-hosted/pull/1800#discussion_r1017148876: always run from the project root and define all paths as relative from there. Should I give that a go?

Reply:
For reference, this was the offending code I removed that causes the behaviour change between tests and an actuall install: https://github.com/getsentry/self-hosted/blob/97a4e5cce4fd74685ff0694ab2f67dd75cf5723f/install/_lib.sh#L11-L16

Reply:
It seems reasonable to run things from the project root and define all paths as relative from there. The primary concern for me is that the absolute path may include details for our error reporting that we don't want to be sending. Otherwise, this seems like a change that's fine with me, so thanks for putting this together.

Reply:
Alright, I changed it so everything is always run from and relative to project root.